tests/foreman/ui/test_repository.py

```diff
@@ -595,6 +595,95 @@ def test_positive_sync_custom_repo_docker(self):
                     self.assertTrue(self.prd_sync_is_ok(repo_name))
 
     @run_only_on('sat')
+    @tier2
+    def test_positive_resynchronize_rpm_repo(self):
+        """Check that repository content is resynced after packages were
+        removed from repository
+
+        @id: dc415563-c9b8-4e3c-9d2a-f4ac251c7d35
+
+        @Assert: Repository has updated non-zero package count
+
+        @CaseLevel: Integration
+
+        @BZ: 1318004
+        """
+        with Session(self.browser) as session:
+            repo = entities.Repository(
```

Comment:
Repo creation is basically specific setup for this test. Also covering of repo creation via UI is not  a purpose of this test, so using API requests is a matter of performance.